packages/website/assets/icons/windows.js

```diff
@@ -0,0 +1,9 @@
+const Windows = () => (
+  <svg id="gui" xmlns="http://www.w3.org/2000/svg" width="25.548" height="24.32" viewBox="0 0 25.548 24.32">
+    <g id="Group_218" data-name="Group 218">
+      <path id="Path_18931" data-name="Path 18931" d="M21.9,6.828h2.387a1.9,1.9,0,0,1,2.083,2.083V23.318A1.877,1.877,0,0,1,24.291,25.4c-5.6,0-11.239.043-16.837,0-1.432,0-2.083-.694-2.126-2.213-.043-1.085,0-2.213,0-3.515-.955,0-1.779.043-2.56,0A1.777,1.777,0,0,1,.858,17.763q-.065-7.355,0-14.711A1.808,1.808,0,0,1,2.9,1.1H19.908c1.345,0,2,.781,2,2.126ZM7.193,10.951c0,3.732.043,7.334-.043,10.935-.043,1.389.521,1.692,1.779,1.692,4.036-.043,8.071,0,12.15-.043,4.036-.043,3.385.477,3.428-3.472.043-2.994,0-6.032,0-9.113Zm12.8-5.728H2.68V15.681c0,2.257.13,2.343,2.647,2V9.128c0-1.519.694-2.213,2.257-2.213,2.864-.043,5.771,0,8.635,0h3.775Z" transform="translate(-0.825 -1.1)" fill="#0008d9"/>
+    </g>
```

Comment:
nit: suprising id attributes. We've used `svgo` previously to handle that job. This may already be part of the workflow as the others so far look clean.

Reply:
SVGs _should_ have all been optimized, and _svgo_ is still included. Will search through and find any unoptimized SVGs (some PNGs might be getting converted to SVGs soon as well, so there may be some new ones soon).